test/specs/helpers.canvas.tests.js

```diff
@@ -21,6 +21,18 @@ describe('Chart.helpers.canvas', function() {
       expect(chart.ctx.clearRect.calls.first().object).toBe(chart.ctx);
       expect(chart.ctx.clearRect.calls.first().args).toEqual([0, 0, 150, 245]);
     });
+
+    it('should not throw error when chart is null', function() {
+      function createAndClearChart() {
+        var chart = acquireChart({}, {
+          canvas: null
+        });
+
+        helpers.clearCanvas(chart.canvas, chart.ctx);
+      }
+
+      expect(createAndClearChart).not.toThrow();
+    });
```

Comment:
This test does not seem to work.
If you don't change `helpers.canvas.ts` and only add this test and expect it to throw it does not throw an error. It seems that passing `null` to the canvas object does not make the chart without a canvas

as you can see here in the test utils package it fills the config with default values when null is passed: https://github.com/chartjs/chartjs-test-utils/blob/master/src/utils.js#L36-L44

![image](https://github.com/chartjs/Chart.js/assets/39033624/ccca2df1-74e2-4efe-90d6-8a1aaadfeafb)
![image](https://github.com/chartjs/Chart.js/assets/39033624/30013a9b-9203-4fbd-add0-7fb468298ca6)

Reply:
Sorry for that and good catch, I believe this should be resolved in 6e1e356